src/web/client/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ComponentState, Component, ChangeEvent } from 'react';
+import { Container, FormControl, TextField, Button, Typography } from '@material-ui/core';
+
+interface Props { }
+
+interface State {
+  username: string;
+  password: string;
+}
+
+export default class Login extends Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      username: "",
+      password: "",
+    };
+  }
+
+  handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    let target: HTMLInputElement = (event.target as HTMLInputElement);
+    let label: string = target.name;
+    let value: string = target.value;
+    this.setState({ [label]: value } as ComponentState);
+  };
+
+  loginSubmit = (): void => {
+    const requestOptions = {
+      method: "POST",
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({username:this.state.username, password:this.state.password})
+    };
+    fetch(`/auth/login`, requestOptions).then(res => {
+      this.setState({username:"" + res.status});

```

Comment:
Were you able to make the request without specifying `http://localhost:8081` ? Because the proxy didn't work for me. :|

Reply:
Worked for me